catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java

```diff
@@ -138,7 +135,7 @@ public void setQuerySources(Set<String> querySources) {
     if (querySources == null) {
       this.querySources.clear();
     } else {
```

Comment:
could make this an if/else if/else instead of nesting the if/else in the top level else